runtime/Go/antlr/input_stream.go

```diff
@@ -94,7 +94,7 @@ func (is *InputStream) GetText(start int, stop int) string {
 
 func (is *InputStream) GetTextFromTokens(start, stop Token) string {
 	if start != nil && stop != nil {
-		return is.GetTextFromInterval(NewInterval(start.GetTokenIndex(), stop.GetTokenIndex()))
+		return is.GetTextFromInterval(NewInterval(start.GetStart(), stop.GetStop()))
```

Comment:
Shouldn't this go in runtime/Go/antlr/***v4/***input_stream.go, not runtime/Go/antlr/input_stream.go?

Reply:
Sorry, it's ok now, I push the new code